packages/indexer/src/index.ts

```diff
@@ -4,13 +4,40 @@ import { Indexer as IndexerRpc } from './indexer.gen'
 
 const fetch = typeof global === 'object' ? global.fetch : window.fetch
 
+export interface SequenceIndexerOpts {
+  JWTAuth?: string
+  ProjectAccessKey?: string
+}
+
 export class SequenceIndexer extends IndexerRpc {
+  public jwtAuth?: string
```

Comment:
I guess we should call it `apiSecretKey`, as this is called "Builder Admin API Secret Keys" in the Builder UI.